5_People/career-operations.md

```diff
@@ -1,38 +1,44 @@
 # Operations Career
 
-You start by sitting at the support desk and answering basic support enquiries by understanding the users' questions and finding solutions for them in our documentation. Then you start taking on billing and similar more sensitive issues, followed by being able to handle technical issues as well. After this point you can switch over to the Technical career or continue into managerial support positions, finally taking over company management.
+You start by sitting at the support desk and answering tech support enquiries by understanding the users' questions and finding solutions. You advance by taking on harder technical issues and communicating regularly with developers. After this point you can switch over to the Technical career or continue into managerial support positions, finally taking over company management.
 
-## Customer Support Specialist
+## Tech Support I
 
-* Handles basic support questions, mostly by knowing where to find answers in our docs.
-* Usually less than 2 years of experience being in Customer Support.
-
-## Customer Support Associate
-
-* Write and improves support documentation.
-* Handles billing issues.
+* Handles a large range of support questions, many of them with technical issues to resolve.
+* Writes and improves support documentation.
```

Comment:
Maybe we can remove `support` to make it more general and less constricting. Documentations are being moved to the devops repo. And since we are now hiring technical people, they might also need to update other related docs.